db/data/20230307105727_create_audits_for_existing_budgets.rb

```diff
@@ -0,0 +1,42 @@
+# Run me with `rails runner db/data/20230307105727_create_audits_for_existing_budgets.rb`
+
+# We are using the Audited Gem to create revisions for Budgets.
+#
+# The "Original" Budget will be the audit that is created when the
+# Budget is first created. However, while this works for new Budgets,
+# we must create an audit for all existing Budgets as well.
+#
+# Audited does not provide an easy way to do this. This uses a private
+# method of the library, then updates the created_at attribute to match the
+# created_at attributes of the Budget itself.
+
+Budget.class_eval do
+  def backfill_audit
+    write_audit(action: "create", audited_changes: audited_attributes,
+      comment: audit_comment)
+
+    Audited::Audit.find_by(auditable_type: "Budget", auditable_id: id).update!(created_at: created_at)
```

Comment:
Just had another thought @CristinaRO Maybe this should be `.update!(created_at: updated_at)` (instead of created_at), so that the "Original" budget is the most recent update of the budget 😕

Reply:
I'm reasonably comfortable with running the migration during a quiet time, but it's up to you!